sdk/react/constants.js

```diff
@@ -27,7 +27,7 @@ var BUTTON_NAMES = { // must match Objective-C code expectations & skin.json.
   // more option buttons
   DISCOVERY: 'discovery',
   QUALITY: 'quality',
-  CLOSED_CAPTIONS: 'closedCaption',
+  AUDIO_AND_CC: 'audioAndCC',
```

Comment:
Is this read correctly by the accessibility screen readers? I'm fearing it won't be understandable.